controllers/socket/lobbyHandlers.go

```diff
@@ -219,6 +220,14 @@ func lobbyJoinHandler(so socketio.Socket) func(string) string {
 			if lob.IsFull() {
 				lob.State = models.LobbyStateReadyingUp
 				lob.Save()
+				go func() {
+					tick := time.After(time.Second * 30)
+					select {
+					case <-tick:
```

Comment:
This will only tick once and because select will block until it hears another tick, the function will never return. That will probably lead to memory leaks or something equally horrible. Just make the function return after saving the lobby.